bash_scrips/intro_to_bash.md

```diff
@@ -0,0 +1,132 @@
+# A brief introduction to bash scripting
+
```

Comment:
I think we should add the author's name and the date below each of our little tutorials.